src/main/kotlin/solve/scene/model/OrderManager.kt

```diff
@@ -0,0 +1,6 @@
+package solve.scene.model
+
+interface OrderManager<T> {
```

Comment:
why manager can not move layers itself? also what about hiding?

Reply:
I use this interface to hide possibility to edit layers order from FrameView. Settings panel will edit the order through the SceneController class, which knows about Scene.